src/transformers/models/seaformer/modeling_seaformer.py

```diff
@@ -0,0 +1,764 @@
+    shape = (input.shape[0],) + (1,) * (input.ndim - 1)  # work with diff dim tensors, not just 2D ConvNets
+    random_tensor = keep_prob + torch.rand(shape, dtype=input.dtype, device=input.device)
+    random_tensor.floor_()  # binarize
+    output = input.div(keep_prob) * random_tensor
+    return output
+
+
+# Copied from transformers.models.convnext.modeling_convnext.ConvNextDropPath with ConvNext->Seaformer
+class SeaformerDropPath(nn.Module):
+    """Drop paths (Stochastic Depth) per sample (when applied in main path of residual blocks)."""
+
+    def __init__(self, drop_prob: Optional[float] = None) -> None:
+        super().__init__()
+        self.drop_prob = drop_prob
+
+    def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
+        return drop_path(hidden_states, self.drop_prob, self.training)
+
+    def extra_repr(self) -> str:
+        return "p={}".format(self.drop_prob)
+    
+
+class SeaformerConv2d_BN(nn.Module):
+    def __init__(self, a, b, ks=1, stride=1, pad=0, dilation=1,
+                 groups=1, bn_weight_init=1, bias=False):
+        super().__init__()
+        self.inp_channel = a
+        self.out_channel = b
+        self.ks = ks
+        self.pad = pad
+        self.stride = stride
+        self.dilation = dilation
+        self.groups = groups
+        self.add_module('c', nn.Conv2d(
+            a, b, ks, stride, pad, dilation, groups, bias=bias))
+        
+        bn = nn.BatchNorm2d(b)
+        nn.init.constant_(bn.weight, bn_weight_init)
+        nn.init.constant_(bn.bias, 0)
+        self.add_module('bn', bn)
+        
+    def forward(self, x):
+        return self.bn(self.c(x))
+
+class SeaformerMLP(nn.Module):
+    """
+    Linear Embedding.
+    """
+    def __init__(self, in_features, hidden_features=None, out_features=None, act_layer=nn.ReLU, drop=0.):
+        super().__init__()
+        out_features = out_features or in_features
+        hidden_features = hidden_features or in_features
+        self.fc1 = SeaformerConv2d_BN(in_features, hidden_features)
+        self.dwconv = nn.Conv2d(hidden_features, hidden_features, 3, 1, 1, bias=True, groups=hidden_features)
+        self.act = act_layer()
+        self.fc2 = SeaformerConv2d_BN(hidden_features, out_features)
+        self.drop = nn.Dropout(drop)
+
+    def forward(self, x):
+        x = self.fc1(x)
+        x = self.dwconv(x)
+        x = self.act(x)
+        x = self.drop(x)
+        x = self.fc2(x)
+        x = self.drop(x)
+        return x
+
+class SeaformerInvertedResidual(nn.Module):
+    def __init__(
+            self,
+            inp: int,
+            oup: int,
+            ks: int,
+            stride: int,
+            expand_ratio: int,
+            activations=None
+    ) -> None:
+        super(SeaformerInvertedResidual, self).__init__()
+        self.stride = stride
+        self.expand_ratio = expand_ratio
+        assert stride in [1, 2]
+
+        if activations is None:
+            activations = nn.ReLU
```

Comment:
Could this be accessed via the configuration by passing config as an argument? (config.activation)